app/main.py

```diff
@@ -1,6 +1,27 @@
+import os
+import requests
+from dotenv import load_dotenv
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports)